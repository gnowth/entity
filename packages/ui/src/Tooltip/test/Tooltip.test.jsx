import 'jest-styled-components';
import renderer from 'react-test-renderer';
import React from 'react';

import UITooltip from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <UITooltip componentProps={{ name: 'apple' }}>
      Test
    </UITooltip>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
