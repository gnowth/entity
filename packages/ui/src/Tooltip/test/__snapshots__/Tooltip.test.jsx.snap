// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Tooltipstyles__Wrapper-e5uewv-0 sc-bdVaJa inYuCB"
>
  <i
    className="icon icon-apple Icon__UIIcon-sc-10so2ko-0 dgxCUQ"
    name="apple"
    onClick={[Function]}
  />
  <div
    className="Tooltipstyles__Popup-e5uewv-1 bJpmwY"
    hidden={true}
  >
    Test
  </div>
</div>
`;
